src/Documentation/Orleans-Configuration-Guide/Client-Configuration.md

```diff
@@ -134,17 +134,42 @@ config.RegisterStreamProvider<SimpleMessageStreamProvider>("SimpleMessagingStrea
         { "PubSubType", "ExplicitGrainBasedAndImplicit" }
     });
 
-// Initialize client using config defined above
-while (!GrainClient.IsInitialized)
+IClusterClient client = null;
+while (true)
 {
     try
     {
-        GrainClient.Initialize(config);
+        // Build a client and then connect it to the cluster.
+        client = new ClientBuilder()
+            .UseConfiguration(config)
+            .ConfigureServices(
+                services =>
+                {
+                    // Services can be provided to the client here. These services are made
+                    // available via dependency injection.
+                    // ConfigureServices can be called multiple times for a single
+                    // ClientBuilder instance.
```

Comment:
maybe add one example here would be better for user to understand, something like:
` services.AddSingleton(yourService);`

Because without looking into public API, it's not clear to me what type of delegate I should feed to `ConfigureServices`